mass_mailing_switzerland/models/mass_mailing_contact.py

```diff
@@ -42,6 +42,34 @@ def get_partner(self, email):
                 ("opt_out", "!=", True)
             ], limit=1).id
 
+    @api.multi
+    def write(self, values):
+        out = super().write(values)
+        # can't be simplified because we are looking for is_email_valid == False
+        # AND is_email_valid is in values (return None otherwise)
+        if values.get('is_email_valid') is False:
+            for mailing_contact in self:
+                # assure is_valid_email as not been reset (compute field might have been trigger)
+                if mailing_contact.is_email_valid:
+                    mailing_contact.is_email_valid = False
```

Comment:
I think the underlying problem is that `is_email_valid` is a compute field and the mailchimp module writes into that field but it's kind of wrong. To fix this better, we should simply convert the field to a regular field, by redefining the field and set the attribute `compute=False`.

Reply:
This change should be directly made into `paid-addons/mailchimp` module (because it's already wrong at this stage)